lib/bibliothecary/configuration.rb

```diff
@@ -10,7 +10,7 @@ class Configuration
     attr_accessor :cabal_parser_host
 
     def initialize
-      @ignored_files = ['.git', 'node_modules', 'bower_components', 'spec/fixtures', 'vendor', 'dist']
+      @ignored_dirs = ['.git', 'node_modules', 'bower_components', 'fixtures', 'vendor', 'dist']
```

Comment:
This isn't quite the same...  as a directory called `foo/fixtures` will now be ignored. I'm trying to decide if I think that matters

Reply:
I think the "right" thing to do might be to remove fixtures entirely and rework the test case.